### PR TITLE
FEATURE: Highlight my mentions and @here/@all

### PR DESCRIPTION
**Pull request by OsamaSayegh**
image

### CODE REVIEW

File: assets/javascripts/discourse/components/tc-message.js [hunk 1]
```diff
@@ -32,6 +32,23 @@ export default Component.extend({
...
+    this._super(...arguments);
+    if (!this.currentUser) {
+      return;
+    }
+    this.element
+      .querySelector(".tc-text")
+      .querySelectorAll(".mention")
+      .forEach((node) => {
+        const mention = node.textContent.trim().substr(1);
+        const highlightable = [this.currentUser.username, "here", "all"];
```
Comment (jjaffeux) [inline assets/javascripts/discourse/components/tc-message.js:45]: maybe have here and all in some constant?

Comment (OsamaSayegh) [reply assets/javascripts/discourse/components/tc-message.js]: sure, done!

**Review by pmusaraj — APPROVED** [review]

**Comment by pmusaraj** [comment]
Unrelated to this PR: in your screenshot that `@here` and `@all` don't get rounded corners.

**Comment by OsamaSayegh** [comment]
Yes, this is because `@here` and `@all` are not styled the same as regular mentions (no background color at all, they're just bold). Not sure if this is intentional or not.

**Pull request merged**